### PR TITLE
Added Serverless support content

### PR DESCRIPTION
**Pull request by abrennan89**
cc @mgencur @bbrowning 

WIP since I will add any additional support info to this same PR, e.g. link to support docs.

### CODE REVIEW

File: modules/serverless-about-collecting-data.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+[id="serverless-about-collecting-data_{context}"]
+= About collecting {ServerlessProductName} data
+
+You can use the `oc adm must-gather` CLI command to collect information about your cluster, including features and objects associated with {ServerlessProductName}.
+
+To collect {ServerlessProductName} data with `must-gather`, you must specify the {ServerlessProductName} image:
+
+----
+$ oc adm must-gather --image=quay.io/openshift-knative/must-gather
```
Comment (mgencur) [inline modules/serverless-about-collecting-data.adoc:13]: AFAICT, the final location of the productized image will be: `registry.redhat.io/openshift-serverless-1/svls-must-gather-rhel8` so this command should be changed to `oc adm must-gather --image=registry.redhat.io/openshift-serverless-1/svls-must-gather-rhel8`

File: serverless/serverless-support.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+include::modules/serverless-getting-support.adoc[leveloffset=+1]
+
+== Collecting {ServerlessProductName} data for Red Hat Support
+
+When opening a support case, it is helpful to provide debugging
+information about your cluster to Red Hat Support.
+
+The `must-gather` tool enables you to collect diagnostic information about your
+{product-title} cluster, including virtual machines and other data related to
```
Comment (mgencur) [inline serverless/serverless-support.adoc:17]:
```suggestion
{product-title} cluster, including data related to
```

Comment (mgencur) [reply serverless/serverless-support.adoc:17]: I'm not aware of virtual machines related to Serverless. This is probably specific to Service Mesh.

File: _topic_map.yml [hunk 3]
```diff
@@ -1626,6 +1626,9 @@ Topics:
 # Release notes
 - Name: Release Notes
   File: serverless-release-notes
+# Support
+- Name: Getting support
```
Comment (mgencur) [inline _topic_map.yml:1630]: I'm not sure about the title. This article doesn't provide a manual how to get support for Serverless. Maybe just "Support" like it is in the Service Mesh docs? These docs will be related to providing information `for` support.

**Comment by abrennan89** [comment]
Thanks @mgencur :slightly_smiling_face: ready for 2nd round reviews now.

File: _topic_map.yml [hunk 4]
```diff
@@ -1627,6 +1627,9 @@ Topics:
 # Release notes
 - Name: Release Notes
   File: serverless-release-notes
+# Support
+# - Name: Support
+#  File: serverless-support
```
Comment (mgencur) [inline _topic_map.yml:1632]: Why is this commented out? A temporary change?

Comment (mgencur) [reply _topic_map.yml:1632]: I'm curious if you plan to rename the section to "Support" or something else.

**Review by mgencur — COMMENTED** [review]
Overall, looks good. But I can't see the rendered version because the topic is commented out.

File: _topic_map.yml (hunk 4 above)
Comment (mgencur) [reply _topic_map.yml:1632]: The file is called `serverless-support` . Not sure where the file is but the original file is still called `serverless-getting-support.adoc`. Maybe it should be renamed?

**Comment by mgencur** [comment]
Alright, after discussion with @abrennan89 it seems this is finished.
QE Approved

**Comment by bergerhoffer** [comment]
This LGTM - @abrennan89 we just need to know the version(s) this is for. Thanks!

**Pull request merged**